MapStatistics.cs

```diff
@@ -146,6 +156,104 @@ public static int UpDown
       }
     }
 
+    public static int MiscConnect
+    {
+      get
+      {
+        var labeled = Project.Current.Elements.OfType<Connection>().Count(p =>
+        {
+          if ((p.EndText != $"") || (p.StartText != $""))
```

Comment:
More of a stylistic issue...but I wouldn't include the $ on these empty strings.  I'd either leave it as "" or better yet, make them String.Empty